### ISSUE TITLE
The hover-over for the menu tile is broken in tN

### ISSUE THREAD

**Issue by cckozie** [unfoldingWord/translationCore#6064]
1.2.0 (1612995) 
This may have been caused by #6000, although it theoretically should not be in this build.
[en_test_1ti_book.zip](https://github.com/unfoldingWord-dev/translationCore/files/3163635/en_test_1ti_book.zip)

This project is fully aligned and has one tN check completed

1. Load the project in wA
2. Edit verse 1:3 to change the word "doctrine" to "theology". This will invalidate the alignment and the tN check.
3. Select a reason code and click Save (The lack of the reset alert is covered in #5941)
4. Return the the Tools page
5. Load the project in tN
6. Note that on verse 1:3 under Assumed Knowledge...  the hover over is broken for both the icons and the extended selection text. (The problem with the selection still being displayed on the menu tile is also covered in #5941)

**Comment by mannycolon** [comment]
Hey @cckozie could please give more details about what's broken. Maybe show some screenshots.

**Comment by mannycolon** [comment]
The icon only shows the tooltip when you hover over it and it has more than one icon to show. For example, if you bookmark the check. 
<img width="239" alt="Screen Shot 2019-05-09 at 4 28 47 PM" src="https://user-images.githubusercontent.com/17072300/57484631-8f28c800-7277-11e9-9d49-0b5e9482740e.png">

<img width="241" alt="Screen Shot 2019-05-09 at 4 28 55 PM" src="https://user-images.githubusercontent.com/17072300/57484642-951ea900-7277-11e9-922f-dfd394b56b39.png">


So, in this case, the icons tooltip isn't expected:

<img width="241" alt="Screen Shot 2019-05-09 at 4 35 54 PM" src="https://user-images.githubusercontent.com/17072300/57485018-9ac8be80-7278-11e9-87f5-35373e941481.png">

**Comment by mannycolon** [comment]
I think the selection on the menu shouldn't be shown in the menu since it was invalidated. Is this what you're seeing as well?

**Comment by cckozie** [comment]
@mannycolon Yes, that is on another issue, #5941
Re: The icon only shows the tooltip when you hover over it and it has more than one icon to show. For example, if you bookmark the check.
Yes, you are correct, but there are really 2 problems here.
1. There should be 2 icons, one for edit and one for invalidated. I'm not sure if that should be on this issue or different one.
2. The text for the check is showing the ellipsis so there should be a hover over for that.
However, I just checked and the extended text hover over does work if the check hasn't been invalidated.

**Comment by cckozie** [comment]
1.2.0 (2fea0fd)
The hover over for icons works correctly in the use case above. However, there is still a problem in this scenario:
1. Load a project in tN
2. Make a selection and click Save. The checkmark is displayed on the menu tile.
3. Click the bookmark switch. The bookmark icon is displayed on the menu tile. Both bookmark and checkmark are displayed in the hover over.
4. Click the Edit verse button and edit the verse so as to invalidate the selection.
5. Choose a reason and click Save. 
The hover over now still displays only the bookmark and checkmark. It should also display the edit icon.
I'm waiting on #5941 and possibly others to see if this is fixed there.

**Comment by cckozie** [comment]
Works as expected in 1.2.0 (cf6fd22)
